tests/components/ozw/test_websocket_api.py

```diff
@@ -56,3 +58,75 @@ async def test_websocket_api(hass, generic_data, hass_ws_client):
     assert result["received_packets"] == 3594
     assert result["received_dup_packets"] == 12
     assert result["received_unsolicited"] == 3546
+
+    # Test node metadata
+    await client.send_json({ID: 8, TYPE: "ozw/node_metadata", NODE_ID: 39})
+    msg = await client.receive_json()
+    result = msg["result"]
+    assert result["metadata"]["ProductPic"] == "images/aeotec/zwa002.png"
+
+
+async def test_refresh_node(hass, generic_data, sent_messages, hass_ws_client):
+    """Test the ozw refresh node api."""
+    receive_message = await setup_ozw(hass, fixture=generic_data)
+    client = await hass_ws_client(hass)
+
+    # Send the refresh_node_info command
+    await client.send_json({ID: 9, TYPE: "ozw/refresh_node_info", NODE_ID: 39})
+    msg = await client.receive_json()
+
+    assert len(sent_messages) == 1
+    assert msg["success"]
+
+    # Receive a mock status update from OZW
+    message = MQTTMessage(
+        topic="OpenZWave/1/node/39/",
+        payload={"NodeID": 39, "NodeQueryStage": "initializing"},
+    )
+    message.encode()
+    receive_message(message)
+
+    # Verify we got expected data on the websocket
+    msg = await client.receive_json()
+    result = msg["event"]
+    assert result["type"] == "node_updated"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)